pkgs/tools/package-management/nix-update/default.nix

```
@@ -1,21 +1,21 @@
 { lib
-, buildPythonApplication
+, python3
 , fetchFromGitHub
 , nix
```

this update adds a nix-prefetch-git as a runtime dependency

```suggestion
, nix
, nix-prefetch-git
```

---

Thanks, fixed.